docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,361 @@
+- [Column management panel with support for pivoting and other features](https://github.com/mui/mui-x/issues/5700)
+- [Pivoting](https://github.com/mui/mui-x/issues/214) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+### Tree View
+
+- [Inline node name editing](https://github.com/mui/mui-x/issues/10112)
+- [Checkbox selection for Simple- and RichTreeView](https://github.com/mui/mui-x/pull/11452)
+- [Drag and drop support for RichTreeView](https://github.com/mui/mui-x/issues/9686)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Charts
+
+- Improved label flexibility and customization
+- [Zoom and navigation](https://github.com/mui/mui-x/issues/12503)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Heatmap](https://github.com/mui/mui-x/issues/7926)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Date and Time Pickers
+
+- [TimeRangePicker](https://github.com/mui/mui-x/issues/4460)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+For a complete overview, please visit the [MUI X roadmap](https://github.com/mui/mui-x/projects/1).
+
+## Long-Term Support
+
+We roll bug fixes, performance enhancements, and other improvements into new releases, so as we introduce MUI X v7, MUI X v6 now is officially in [Long-Term Support (LTS)](https://mui.com/x/introduction/support/#long-term-support-lts). This means only critical bug fixes and security updates will be patched to MUI X v6.
+
+Support for MUI X v5 is officially **discontinued**, and we encourage users to migrate to v6 and then to v7 to benefit from enhanced features, performance improvements, and ongoing support.
+
+## Migration guide
+
+We fully understand that updating a dependency library can sometimes be daunting.
+To smooth the transition to the new version, we have documented all breaking changes in our migration guides:
+
+- [Data Grid](https://mui.com/x/migration/migration-data-grid-v6/)
+- [Date and Time Pickers](https://mui.com/x/migration/migration-pickers-v6/)
+- [Charts](https://mui.com/x/migration/migration-charts-v6/)
+- [Tree View](https://mui.com/x/migration/migration-tree-view-v6/)
+
+These component-specific guides are reference pages designed to assist you if you encounter any challenges while updating to the new major version.
+It's our way of making the upgrade process more manageable and less burdensome for you.
+However, should you need further assistance, please don't hesitate to contact us and send your feedback.
+
+## How to get involved
+
+Your feedback played a crucial role in shaping MUI X, and we always appreciate hearing from you.
+To continue enhancing our product, we invite you to share your pain points and challenges with us by:
+
+- [Giving us a user interview](https://forms.gle/vsBv6CLPz9h57xg8A).
```

Comment:
```suggestion
- [Talking to us on a call](https://forms.gle/vsBv6CLPz9h57xg8A).
```

I think "Giving us a user interview" is a bit of a weird saying. Like, "to give a user interview" is just to "talk to the makers and give feedback". I'd probably go for something along these lines.